### PR TITLE
Java version of the forward euler algorithm

### PR DESCRIPTION
**Pull request by William103**
This is just a simple version of the algorithm in java. In the text it mentioned using the equation y'=5y, but all the other examples use y'=y-3y and it works out better that way so I used that equation. Also, this is my first attempt at a contribution, so if I'm doing anything wrong or could even just do something better, please let me know.

### CODE REVIEW

**Comment by leios** [comment]
Hey, we appreciate the submission, but are somehow missing expertise in Java, so I'll get to this soon, but was waiting to see if anyone who knew Java more than I do wanted to review it first.

File: contents/forward_euler_method/code/java/euler.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Submitted by William103
+
+public class ForwardEuler {
+    public static void main(String[] args) {
+		int n = 100;
+		double dt = 0.01;
+		double initial = 1;
+		double threshold = 0.01;
```
[the remark this answers is not in the document]
Comment (leios) [reply contents/forward_euler_method/code/java/euler.java]: Thanks for that! Really appreciate it!

**Comment by Yordrar** [comment]
I just saw that there is an older PR about this same algorithm which I also reviewed, please check that no one worked on the algorithm you are implementing

**Comment by leios** [comment]
Hey, I apologize for the inconvenience here, we just realized there was / is an earlier PR #387 that does this implementation. We usually go on a first-come, first-serve basis here, but I'll keep this PR up in the case that the other PR creator doesn't respond.

**Comment by William103** [comment]
Ok, I made the changes Yordrar suggested. Sorry that it's a duplicate. I'll triple check next time.

**Comment by julianschacher** [comment]
I'll close it for now. If #387 doesn't get merged or fails somehow, we can reopen that again.

**Pull request closed without merging**